tests/integration_tests/modules/test_power_state_change.py

```diff
@@ -0,0 +1,90 @@
+"""Integration test of the cc_power_state_change module.
+
+Test that the power state config options work as expected.
+"""
+
+import pytest
+import time
```

Comment:
A nit, which I'll apply before merging:

```suggestion
import time

import pytest
```

(pytest is a third-party module.)